src/uct/tcp/tcp_ep.c

```diff
@@ -136,28 +136,27 @@ static void uct_tcp_ep_close_fd(int *fd_p)
     }
 }
 
-/* Must be called with `iface::worker::async` blocked */
-static unsigned uct_tcp_ep_in_iface(uct_tcp_ep_t *ep)
+unsigned uct_tcp_ep_peer_addr_is_equal(const uct_tcp_ep_t *ep,
```

Comment:
looks like this func is not needed, `uct_tcp_ep_peer_addr_to_itself` would be enough

Reply:
nice catch, done